javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+var bodyBackGround = document.querySelector('body');
+var light = document.getElementById('bulb');
+
+var lightIsOn = true;
+light.addEventListener('click', function (e) {
+  lightIsOn = !lightIsOn;
+  if (!lightIsOn) {
+    light.classList.add('light-off');
+    bodyBackGround.style.backgroundColor = 'rgb(17,17,17)';
```

Comment:
Instead of setting styling on the element directly like this, you should prepare a class that you can add and remove.